src/extension.ts

```diff
@@ -106,7 +106,205 @@ export function activate(context: ExtensionContext) {
+	         return buildItems(getEnvNames(), range);
+	       }
+	     },
+	     ...IDENT_CHARS, '.', '[', '"', '\'', '('
+	   )
+	 );
+
+	 const pyProviders = ['python'].map(lang =>
+	   languages.registerCompletionItemProvider(
+	     lang,
+	     {
+	       provideCompletionItems(document: TextDocument, position: Position): CompletionItem[] | undefined {
+	         const lineText = document.lineAt(position.line).text;
+	         const prefix = lineText.substring(0, position.character);
+	         const startChar = detectPyStartChar(prefix);
+	         if (startChar === undefined) { return undefined; }
+	         const range = new Range(new Position(position.line, startChar), position);
+	         return buildItems(getEnvNames(), range);
+	       }
+	     },
+	     ...IDENT_CHARS, '[', '"', '\'', '('
+	   )
+	 );
+
+	 // HOVER: mostra o valor ao passar o mouse no nome da env var
+	 function isJsEnvContext(left: string): boolean {
+	   if (/process\.env\.\s*$/.test(left)) { return true; }
+	   if (/process\.env\[\s*(['"])\s*$/i.test(left)) { return true; }
+	   return false;
+	 }
+
+	 function isPyEnvContext(left: string): boolean {
+	   if (/os\.environ\[\s*(['"])\s*$/i.test(left)) { return true; }
+	   if (/os\.environ\.get\(\s*(['"])\s*$/i.test(left)) { return true; }
+	   if (/os\.getenv\(\s*(['"])\s*$/i.test(left)) { return true; }
+	   return false;
+	 }
+
+	 const wordRegex = /[A-Za-z_][A-Za-z0-9_]*/;
+
+	 const jsHoverProviders = ['javascript', 'javascriptreact', 'typescript', 'typescriptreact'].map(lang =>
+	   languages.registerHoverProvider(lang, {
+	     provideHover(document: TextDocument, position: Position): Hover | undefined {
+	       const range = document.getWordRangeAtPosition(position, wordRegex);
+	       if (!range) { return undefined; }
+	       const word = document.getText(range);
+	       const lineText = document.lineAt(position.line).text;
+	       const left = lineText.substring(0, range.start.character);
+	       if (!isJsEnvContext(left)) { return undefined; }
+	       const val = getEnvValue(word);
+	       if (val === undefined) { return undefined; }
+	       const md = new MarkdownString();
+	       md.appendText(`${word}=${String(val)}`);
+	       md.isTrusted = false;
+	       return new Hover(md, range);
+	     }
+	   })
+	 );
+
+	 const pyHoverProviders = ['python'].map(lang =>
+	   languages.registerHoverProvider(lang, {
+	     provideHover(document: TextDocument, position: Position): Hover | undefined {
+	       const range = document.getWordRangeAtPosition(position, wordRegex);
+	       if (!range) { return undefined; }
+	       const word = document.getText(range);
+	       const lineText = document.lineAt(position.line).text;
+	       const left = lineText.substring(0, range.start.character);
+	       if (!isPyEnvContext(left)) { return undefined; }
+	       const val = getEnvValue(word);
+	       if (val === undefined) { return undefined; }
```

Comment:
Quick question, if a workspace have more than one `.env` files (let's say `.env.local` and `.env.production`) both contain the same variable keys. Upon hovering process.env.[some-var], which environment values get displayed? local or production or both?